Turf.podspec

```diff
@@ -24,6 +24,9 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = "10.13"
   s.tvos.deployment_target = "11.0"
   s.watchos.deployment_target = "4.0"
+  # CocoaPods doesn't support releasing of visionOS pods yet, need to wait for v1.15.0 release of CocoaPods
+  # with this fix https://github.com/CocoaPods/CocoaPods/pull/12159.
+  # s.visionos.deployment_target = "1.0"
```

Comment:
Having this fails the `pod spec lint` with the following error:
```
    - ERROR | [visionOS] unknown: Encountered an unknown error (Could not find a `visionos` simulator (valid values: ios, tvos, watchos, xros). Ensure that Xcode -> Window -> Devices has at least one `visionos` simulator listed or otherwise add one.) during validation.
```

I think it's fine not to support VisionOS for CocoaPods distribution for a while.

Reply:
Can we use the `main` branch of CocoaPods for releases for now? Not sure if it will break integration for customers with pre-1.15 CocoaPods

Reply:
They say it doesn't work still https://github.com/CocoaPods/CocoaPods/pull/12159#issuecomment-1897118199

If nobody fixes it and I have time, I'll try to fix it myself.